microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -19,6 +19,18 @@ On Windows 11, widgets appear in the widgets board which you open from the left
 In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and handle user interactions within them.
 
 
+<!-- ------------------------------ -->
+#### Requires developing a PWA as a widget
```

Comment:
You don't technically develop a PWA _as_ a widget either. A PWA can _support_ a widget, just like a Win32 app can support a widget.

The intent of this section is to call out that you can't just take a PWA and shove it into the widget dashboard like you can with the Edge Sidebar. The web developer needs to build a custom Widget experience that is appropriate for the Widget Host, which currently is the Win11 Widgets Board, but there may be other hosts in the future. The Win11 Widgets board requires that Widgets are built with Adaptive Card templates instead of HTML and JavaScript, so it would have to be designed separately from the rest of the app's UI.

Maybe this should be "Requires building a custom widget for the PWA"? Although I'm not sure if that relays the intent either.

Reply:
added that info, changed heading